lib/pmdtester/builders/pmd_report_builder.rb

```diff
@@ -194,5 +195,16 @@ def saved_distro_path(build_sha)
     def wd_has_dirty_git_changes
       !Cmd.execute('git status --porcelain').empty?
     end
+
+    def should_use_long_cli_options
+      logger.debug "PMD Version: #{@pmd_version}"
+      m = /(\d+)\.(\d+)\.(\d+).*/.match(@pmd_version)
+      major = m[1]
+      minor = m[2]
+      patch = m[3]
+      result = major.to_i >= 6 && minor.to_i >= 41
```

Comment:
this will fail with PMD 7.0.0 as `inor.to_i >= 41` will be false

Reply:
you're right :+1: